recipes/atlas-schema/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "atlas-schema" %}
+{% set version = "0.1.0" %}
+{% set python_min = "3.9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/atlas_schema-{{ version }}.tar.gz
+  sha256: 50f8fd29e958f00ff5fe8b0f2235e281952e5391e893bb38e61c2c50751b7766
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
```

Comment:
We had minor conda-build breakages where it was. I really don't care about such small details that people feel so strongly. I have zero stake on this package. Go for it, if you ever get pip packages in your env, you know why.

Reply:
Sorry, I think I came across in the wrong manner. I thought that this problem only affected packages that had multiple build outputs, and that for all other situations this had gotten patched so that the build tool set all the flags that were required.

@kratsg, as I don't fully understand things, you should apply the `--no-deps --no-build-isolation` patch back.

Reply:
The reason why we have that set on grayskull is b/c we chose to be explicit and avoid regression on tools downstream. We could set those flags in conda-forge but the pip reversed logic and yet another hidden layer of options, would make this awkward. IMO, we should be explicit here to ensure the users know that is happening. Not long ago some people came to me surprised that we use no build isolation in conda-forge as if it was new... It isn't.

Reply:
> IMO, we should be explicit here to ensure the users know that is happening. Not long ago some people came to me surprised that we use no build isolation in conda-forge as if it was new... It isn't.

Just for context, I always assumed that the fact that `grayskull` had this recommendation meant that `conda-forge` **wasn't** using `--no-build-isolation` as my assumption is that "If I have to specify something as the user then this means that it is my responsibility and that it isn't being dealt with elsewhere."

So I took away the opposite of the intended message.